cmd/thanos/rule.go

```diff
@@ -476,8 +479,9 @@ func runRule(
 				signal.Notify(c, syscall.SIGHUP)
 				select {
 				case <-c:
+					rc := make(chan error, 1)
 					select {
-					case reload <- struct{}{}:
+					case reload <- rc:
```

Comment:
Let's log error for this case?

Reply:
@bwplotka  I've committed my change, please help to review. Thanks.